### DISCUSSION TITLE
ютуб работает только первые 5-6 минут

### DISCUSSION THREAD

**Discussion by lopatin531** [bol-van/zapret#795] [Q&A]
Поставил zapret на роутер d-link dr-853 на openwrt. Всё работает прекрасно, всё открывается. Также поставил на xiaomi mi 3 тоже на openwrt. Работает тоже прекрасно. Решил его поставить в другую квартиру. Там у меня другой провайдер. После установки ютуб работает как положено первые 5-6 минут, потом перестаёт грузить видео (поиск, комментарии, всё работает, а видео не грузит). Пробовал различные настройки, картина не меняется. Смущает, что сначала всё отлично работает, а потом отваливается ютуб (инстаграм и прочее продолжают работать). Как быть?

**Comment by numbereleven-a** [comment]
подбирать другую страту не пробовал?

**Reply by lopatin531** [reply]
а как же, перепробовал. те, что работают — ведут себя одинаково. ютуб работает пять минут, потом отваливается, а инстаграм продолжает отлично работать.

**Reply by bol-van** [reply]
Надо изучать вопрос по F12 и и проверять CURLом проблемные домены (GGC ?)
Еще шарк может понадобиться. Если действительно идет TCP, и застревает после переданных мегабайтов, что само по себе странно.
Может быть еще проблема в quic. Его можно попробовать отлключить в броузере и посмотреть изменилось ли что-то

**Reply by lopatin531** [reply]
была бы проблема в quick — телефон или телевизор ютуб бы грузил, а не грузят. 
Провайдер lovit (москва, дома ПИК, их карманный провайдер)... не думаю, что у них что-то особо хитрое есть.
